apps/tests/devapps/instance_discovery_disabled_sample.go

```diff
@@ -0,0 +1,105 @@
+package main
```

Comment:
This file needs a copyright header. Taking a step back though, what's this file for? The user API seems simple enough to not benefit from samples, and the functions here seem to require additional configuration to serve as tests.

Reply:
I think previously sarathy and I thought that this was meant to be the test code, so we added this sample to run. Can remove if you want.

Reply:
Sounds good to me; the new option has unit tests and better documentation elsewhere.

Reply:
I meant I'm in favor of deleting this file because it lacks purpose

Reply:
Deleted